src/scripts/settings/webSettings.js

```diff
@@ -18,11 +18,29 @@ function getDefaultConfig() {
     });
 }
 
-export function enableMultiServer() {
+export function getMultiServer() {
     return getConfig().then(config => {
         return config.multiserver;
     }).catch(error => {
-        console.log('cannot get web config:', error);
+        console.log('cannot get web config: ', error);
```

Comment:
Does JavaScript automatically insert the space here?

Reply:
```
console.log('t:', window);
t: Window {parent: Window, opener: null, top: Window, length: 0, frames: Window, …}

console.log('t: ', window);
t:  Window {parent: Window, opener: null, top: Window, length: 0, frames: Window, …}
```

Reply:
Looks like it does, thanks.